### PR TITLE
Changes rails-admin gem for active-admin

### PR DESCRIPTION
**Pull request by santiagogabrielcastillo**

### CODE REVIEW

File: app/admin/categories.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  #
+  # permit_params :name
+  #
+  # or
+  #
+  # permit_params do
+  #   permitted = [:name]
+  #   permitted << :other if params[:action] == 'create' && current_user.admin?
+  #   permitted
+  # end
```
Comment (jpbalarini) [inline app/admin/categories.rb]: let's remove autogenerated comments if we are not using them

File: app/admin/dashboard.rb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    #       end
+    #     end
+    #   end
+
+    #   column do
+    #     panel "Info" do
+    #       para "Welcome to ActiveAdmin."
+    #     end
+    #   end
+    # end
```
Comment (jpbalarini) [inline app/admin/dashboard.rb]: same here

File: app/admin/orders.rb [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  #
+  # permit_params :service_id, :user_id, :guests, :delivery, :address, :apartment, :date, :from, :to, :comments, :paid
+  #
+  # or
+  #
+  # permit_params do
+  #   permitted = [:service_id, :user_id, :guests, :delivery, :address, :apartment, :date, :from, :to, :comments, :paid]
+  #   permitted << :other if params[:action] == 'create' && current_user.admin?
+  #   permitted
+  # end
```
Comment (jpbalarini) [inline app/admin/orders.rb]: and here

File: app/admin/providers.rb [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+  #
+  permit_params :name, :address, :description, :delivery, :schedule, category_ids: []
+  #
+  # or
+  #
+  # permit_params do
+  #   permitted = [:name, :address, :description, :delivery, :schedule]
+  #   permitted << :other if params[:action] == 'create' && current_user.admin?
+  #   permitted
+  # end
```
Comment (jpbalarini) [inline app/admin/providers.rb]: same here

File: app/admin/services.rb [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+  #
+  # permit_params :name, :description, :cost, :provider_id
+  #
+  # or
+  #
+  # permit_params do
+  #   permitted = [:name, :description, :cost, :provider_id]
+  #   permitted << :other if params[:action] == 'create' && current_user.admin?
+  #   permitted
+  # end
```
Comment (jpbalarini) [inline app/admin/services.rb]: here

File: app/admin/users.rb [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+  # permit_params :email, :encrypted_password, :reset_password_token, :reset_password_sent_at, :remember_created_at, :name, :phone, :admin
+  #
+  # or
+  #
+  # permit_params do
+  #   permitted = [:email, :encrypted_password, :reset_password_token, :reset_password_sent_at, :remember_created_at, :name, :phone, :admin]
+  #   permitted << :other if params[:action] == 'create' && current_user.admin?
+  #   permitted
+  # end
+
```
Comment (jpbalarini) [inline app/admin/users.rb]: same

**Review by jpbalarini — COMMENTED** [review]
Looking great! Awesome work 🚀  
Check my comments and then merge it

**Comment by santiagogabrielcastillo** [comment]
Done!

**Pull request merged**